kymatio/backend/numpy_backend.py

```diff
@@ -1,5 +1,11 @@
 import numpy as np
 
+def input_checks(x):
+    if x is None:
+        raise TypeError('The input should be not empty')
+
+    if not x.flags['C_CONTIGUOUS'] or x is None:
+        raise RuntimeError('The input must be contiguous.')
```

Comment:
Not needed, me thinks. I've never encountered an error related to contiguity in NumPy ever. Torch, yes.

Reply:
Additionally we already checked if the input was empty previously.

Reply:
Agreed. No need for this in NP.